### PR TITLE
Feature/test branch

### PR DESCRIPTION
**Pull request by Yojuks**
Add index 
Change readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-Hello
+Hello World!!!
```
Comment (romawht22) [inline README.md:1]: Error

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    Доброго вечора, ми з України!
+  </body>
+</html>
```
Comment (romawht22) [inline index.html:12]: wrong markup

**Review by romawht22 — CHANGES_REQUESTED** [review]
redo some things

File: index.html (hunk 2 above)
Comment (Yojuks) [reply index.html]: Done

File: README.md (hunk 1 above)
Comment (Yojuks) [reply README.md:1]: Done

**Review by romawht22 — APPROVED** [review]
Bug fixed

**Pull request merged**